.github/CODEOWNERS

```diff
@@ -1308,6 +1308,9 @@ x-pack/test_serverless/**/test_suites/observability/ai_assistant @elastic/obs-ai
 /.devcontainer/ @elastic/kibana-operations
 
 # Appex QA
+/x-pack/test_serverless/README.md @elastic/appex-qa
+/x-pack/test_serverless/api_integration/ftr_provider_context.d.ts @elastic/appex-qa
+/x-pack/test_serverless/api_integration/test_suites/common/README.md @elastic/appex-qa
```

Comment:
Do we consider `README` a test file?

Reply:
If we change [the following wording](https://github.com/elastic/kibana/issues/194551#issue-2558903444), we could get away with not including this file:

```
Add owners for all files in the test directories ...
```

But what do we gain from having readme files not included in this effort?

Oh perhaps we can create an exemption list so the the pr check (mentioned [here](https://github.com/elastic/kibana/issues/192979#issue-2527935670)) does not flag certain files?